### PR TITLE
One pr for issue 180

### PR DESCRIPTION
**Pull request by HonkingGoose**
Hi @deniszholob,

I've taken my changes that I linked to #180, and merged the lot, and rebased on your master branch. This should be okay now.

Note that this pull-request **does not contain** the changes from #184. Do you want that to be included too?

I hope this is what you wanted. :) If not let me know and I'll do some more rebasing... :smile: 

HonkingGoose

### CODE REVIEW

**Comment by deniszholob** [comment]
@HonkingGoose Yeah include #184 as well :)

File: src/app/views/cheat-sheets/basic-power/basic-power.component.html [hunk 1]
```diff
@@ -107,6 +107,7 @@ <h4 class="card-title">Solar Power</h4>
             and during the night, those accumulators release their charge to power your factory.
         </li>
         <li>Place accumulators until they can keep your factorio powered through the night</li>
+        <li>Add some extra accumulators to account for bursty consumption, such as firing a lot of lasers.</li>
```
Comment (deniszholob) [inline src/app/views/cheat-sheets/basic-power/basic-power.component.html]: bursty -> burst

**Comment by HonkingGoose** [comment]
Okay will include #184 too. I'll go merge and rebase that too.

**Comment by HonkingGoose** [comment]
Well one force-push later, we have 3 nice commits, instead of a unholy mess of pull-requests with extra commits.

I only saw your edit just now...
>And feel free to make it just 1 commit too if you want as its all simple changes related to 1 ticket (idc personally, ill squash it all on merge either way).

So do you want another force-push making this just one commit? 
Or do you want to do Github's "squash and merge" yourself?

**Comment by deniszholob** [comment]
Merged, thanks for contributing!

**Pull request merged**